src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -1206,7 +1206,7 @@ bool QgsPostgresProvider::loadFields()
 
 void QgsPostgresProvider::setEditorWidgets()
 {
-  if ( !QgsPostgresUtils::tableExists( connectionRO(), EDITOR_WIDGET_STYLES_TABLE ) )
+  if ( !QgsPostgresUtils::tableExists( connectionRO(), QStringLiteral( "public" ), EDITOR_WIDGET_STYLES_TABLE ) )
```

Comment:
There's potentially a regression here -- the existing code supports these tables being placed in a default schema which is NOT "public". I think the tableExists/columnExists methods should support empty string for schema to use the current logic, which doesn't check the schema.

Reply:
Ok, I did not realize that. Should be fixed as requested.